develop/smart-contracts/evm-toolkit/libraries/ethers-js.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Ethers.js
+description: Learn how to interact with the Asset Hub chain using Ethers.js, compiling and deploying Solidity contracts, and interacting with deployed smart contracts.
```

Comment:
```suggestion
title: Deploy Contracts to Asset Hub with Ethers.js
description: Learn how to interact with the Asset Hub chain using Ethers.js, from compiling and deploying Solidity contracts to interacting with deployed smart contracts.
```

We need to be more descriptive for SEO purposes. Just Ethers.js will get lost in all the other results specific to ethers